src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/images/native_otlp.svg

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
image looks great in both light and dark mode, doesn't bleed into the right side either :)

I did notice that on the "OpenTelemetry quick start" doc, the alternate text reads "native_otlp.png". Consider changing this to be more meaningful

Reply:
@kaileyhaynes, another good catch! I just fixed this.